### PR TITLE
APIS-4015 Added proper README and run_local_with_dependencies script.

### PR DESCRIPTION
**Pull request by FluentSynergyDW**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,74 @@
...
+* `GET http://localhost:9625/nino/:nino/tax-year/:taxYear` (taxYear is in the format YYYY-YY)
+* `Authorization` header set to the bearer token that you obtained from Authority Wizard
+* `Accept` header set to `application/vnd.hmrc.1.0+json`
+
+This will return a `404 Not Found` response because you have not yet set up any
+data for that NINO and tax year.
+
+To set up some data, call the PAYE DES Stub:
+* `POST http://localhost:9689/paye/nino/:nino/tax-year/:taxYear` (taxYear is in the format YYYY-YY)
+* `Content-Type` set to `application/json`
```
Comment (rose-garratt) [inline README.md]: Maybe use the word header after `Content-Type` and `Accept` to be explicit and consistent with the previous instructions.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+To call the Individual PAYE API for the NINO that you specified in Authority Wizard:
+* `GET http://localhost:9625/nino/:nino/tax-year/:taxYear` (taxYear is in the format YYYY-YY)
+* `Authorization` header set to the bearer token that you obtained from Authority Wizard
+* `Accept` header set to `application/vnd.hmrc.1.0+json`
+
+This will return a `404 Not Found` response because you have not yet set up any
+data for that NINO and tax year.
+
+To set up some data, call the PAYE DES Stub:
+* `POST http://localhost:9689/paye/nino/:nino/tax-year/:taxYear` (taxYear is in the format YYYY-YY)
```
Comment (rose-garratt) [inline README.md]: Reference the documentation instead of detailing here as this is public and API docs are private

File: README.md [hunk 3]
```diff
@@ -1,10 +1,74 @@
...
+* `Content-Type` set to `application/json`
+* `Accept` set to `application/vnd.hmrc.2.0+json` (note the 2.0)
+* Raw body set to `{"scenario": "EVERYTHING"}`
```
Comment (rose-garratt) [inline README.md]: Reference the docs instead of detailing here as this is public and the api docs are private

File: README.md [hunk 4]
```diff
@@ -1,10 +1,74 @@
...
+  because you aren't running anything on the redirect URL that you specified.
+* Go to the [Authority Wizard Session](http://localhost:9949/auth-login-stub/session) page
+* Copy the value of the `authToken`. This is the bearer token that you will use to call the API.
+
+To call the Individual PAYE API for the NINO that you specified in Authority Wizard:
+* `GET http://localhost:9625/nino/:nino/tax-year/:taxYear` (taxYear is in the format YYYY-YY)
+* `Authorization` header set to the bearer token that you obtained from Authority Wizard
+* `Accept` header set to `application/vnd.hmrc.1.0+json`
+
+This will return a `404 Not Found` response because you have not yet set up any
```
Comment (rose-garratt) [inline README.md]: Consider moving the data setup to before this call

Comment (FluentSynergyDW) [reply README.md]: Added "If you ..."

**Pull request merged**